libraries/dialogs/src/main/java/com/trendyol/uicomponents/dialogs/list/DialogListItemViewState.kt

```diff
@@ -0,0 +1,18 @@
+package com.trendyol.uicomponents.dialogs.list
+
+import android.text.SpannableString
+import android.text.Spanned
+import androidx.core.text.HtmlCompat
+
+data class DialogListItemViewState(
+    val name: CharSequence,
+    val showAsHtml: Boolean,
+    val isChecked: Boolean
+) {
+
+    fun getText(): Spanned = if (showAsHtml) {
```

Comment:
The return type of the function can be replaced with CharSequence

Reply:
Same as above, `HtmlCompat.fromHtml()` returns Spanned interface.